config-bootstrapper/src/main/resources/configs/config-bootstrapper/entity-service/hypertrace-core/1_entityrelationshiptypes.conf

```diff
@@ -3,13 +3,15 @@ name = entityrelationshiptypes
 commands = [
   {
     upgrade : [
-      {
-        type : EntityRelationshipType,
-        action : DELETE,
-        data : {
-          filter = {}
+        {
+          type : EntityRelationshipType,
+          action : DELETE,
+          data : {
+            tenantId = __default
```

Comment:
Is this needed @samarth-gupta-traceable? CB might be using root user context for the relationships.

Reply:
@buchi-busireddy  Do you mean the context might change from the current way these configs are created ?
I copied the file from existing `entity-service` repo in https://github.com/hypertrace/entity-service/blob/master/helm/configs/1_entityrelationshiptypes.conf 
Now also these conf are picked up by CB once they are placed as configmaps during entity-service deployment.

Reply:
Then I think it's a bug there itself. Just remove `tenantId` row here and things should still work, by using `__root` tenantId.